### PR TITLE
Ana/4110 kusama ux improvement ideas

### LINKED ISSUE #4110: kusama ux improvement ideas
1) signed in with stash only account
Screen Shot 2020-05-21 at 6 21 45 PM

`To: --` probably doesn’t even need to be on the modal. let’s v-if it!

`From` styling is broken. i think we should also stay consistent and hide all the characters in the address except for the first 4 and last 4 on this screen.

`Amount` should not say "(Optional)" because it's not for stash only addresses

2) signed in with controller only account
Screen Shot 2020-05-21 at 6 24 18 PM
- let's change this message to something like “controller addresses don’t “have” validators — use your stash account to see your validator portfolio”"
- let's add the "stake" button beside the "send" button (charlie has designs for this). this should appear for stash+controller and controller (but not stash only)

Screen Shot 2020-05-21 at 6 25 18 PM
- let's hide the `amount` field
- `From` doesn't need to be a dropdown

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4110 

**Description:**

Apply UX improvements that Jordan suggested except the ones I commented on.

I have some doubts on the "To" on Undelegation, but seems in Polkadot it just doesn't make any sense the concept the Restake, so it is cool.

Screenshots:

Stash Stake

Stask Unstake

Controller Stake

Controller Unstake 

Also fixes this:

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
Remove From in Stake+Stash?

**Comment by Bitcoinera** [comment]
Let's merge! 💃

File: src/ActionModal/components/DelegationModal.vue [hunk 1]
```diff
@@ -81,6 +81,7 @@
     </TmFormGroup>
 
     <TmFormGroup
+      v-if="!(session.addressRole === `stash`)"
```
Comment (faboweb) [inline src/ActionModal/components/DelegationModal.vue:84]:
```suggestion
      v-if="!(session.addressRole === `stash`)"
```
```suggestion
      v-if="session.addressRole !== `stash`"
```

Comment (Bitcoinera) [reply src/ActionModal/components/DelegationModal.vue:84]: But you were the one saying that positive was always easier to understand xD

Comment (faboweb) [reply src/ActionModal/components/DelegationModal.vue:84]: that would be if you create a variable `notAStash` vs a variable `isStash`

Comment (Bitcoinera) [reply src/ActionModal/components/DelegationModal.vue]: Done

**Comment by mariopino** [comment]
Love the cleaner behavior!

**Pull request merged**